path_finding.asciidoc

```diff
@@ -223,6 +222,49 @@ Once we attempt to send an HTLC we can learn more about channel balances: if we
 You may be thinking "What's the point of learning from a successful HTLC?" After all, if it succeeded we're "done". But consider that we may be sending one part of a multi-part payment. We also may be sending other single-part payments within a short time. Anything we learn about liquidity is useful for the next attempt!
 ====
 
+==== Liquidity uncertainty and probability
+
+To quantify the uncertainty of a channel's liquidity, we can apply probability theory. A basic model of the probability of payment delivery will lead to some rather obvious, but important, conclusions:
+
+* Smaller payments have a better chance of successful delivery across a path.
+
+* Larger capacity channels will give us a better chance of payment delivery for a specific amount.
+
+* The more channels (hops), the lower the chance of success.
+
+While these may be obvious, they have important implications, especially for the use of Multi-Path Payments (see <<mpp>>). The math is not difficult to follow.
+
+Let's use probability theory to see how we arrived at these conclusions.
+
+First, let's posit that a channel with capacity +c+ has liquidity on one side with an unknown value in the range of +(0, c)+ or "range between 0 and c". For example, if the capacity is +5+, then the liquidity will be in the range +(0, 5)+. Now, from this we see that if we want to send 5 satoshis, our chance of success is only 1 in 6 (16.66%), because we will only succeed if the liquidity is exactly 5.
+
+More simply, if the possible values for the liquidity are 0,1,2,3,4,5 only one of those six possible values will be sufficient to send our payment. To continue this example, if our payment amount was 3, then we would succeed if the liquidity was 3, 4, or 5. So our chances of success are 3 in 6 (50%). Expressed in math, the success probability function for a single channel is:
+
+latexmath:[P_c(a) = (c + 1 - a) / (c + 1)]
+
+where +a+ is the amount and +c+ is the capacity
+
+From the equation we see that if the amount is close to 0, the probability is close to 1 whereas if the amount exceeds the capacity, the probability is zero.
+
+In other words: "Smaller payments have a better chance of successful delivery" or "Larger capacity channels give us better chances of delivery for a specific amount" and "You can't send a payment on a channel with insufficient capacity".
```

Comment:
it's a bit redundant with the three points at the beginning of the section but I think it is very fine to have that redundancy here.